lib/cocoapods/generator/xcconfig/pod_xcconfig.rb

```diff
@@ -45,12 +45,14 @@ def generate
           search_paths = target_search_paths.concat(sandbox_search_paths).uniq
 
           config = {
+            'FRAMEWORK_SEARCH_PATHS' => '$(inherited) ',
```

Comment:
Why do you have these `FRAMEWORK_SEARCH_PATHS` and `LIBRARY_SEARCH_PATHS` here overwritten with `$(inherited) `? On my branch these values are kept anyway when these are overwritten and as long as these aren't overwritten, they don't need to be explicitly set. I ask because somehow this causes double occurrences of `$(inherited)`.